### PR TITLE
Report "Activity's work slips" 3 sections

### PR DESCRIPTION
**Pull request by eilst**

### CODE REVIEW

File: project_event/report/project_task_activity_report.py [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+
+    title = _("Activity's work slips")
+    print_date_text = _("Print Date")
+    activity_label = _("Activity: ")
+    date_label = _("Date: ")
+    start_hour_label = _("Hour: ")
+    end_hour_label = _("Expected End Time: ")
+    number_spectators_label = _("Number of spectators: ")
+    client_label = _("Client: ")
+    contact_label = _("Contact 1 :")
```
Comment (rimbendhaou) [inline project_event/report/project_task_activity_report.py]: why contact 1?

File: project_event/models/models.py [hunk 2]
```diff
@@ -21,3 +23,68 @@ def get_formview_id(self, access_uid=None):
 
 
 BaseModel.get_formview_id = get_formview_id
+
```
Comment (rimbendhaou) [inline project_event/models/models.py:26]: PEP8 expected two blank lines, found1

File: project_event/models/models.py [hunk 3]
```diff
@@ -21,3 +23,68 @@ def get_formview_id(self, access_uid=None):
...
+                res['string'] = res['name']
+            resreport = clear_views_without_print_report(self, view_id, resreport)
+            result['toolbar'] = {
+                'print': resreport,
+                'action': resaction,
+                'relate': resrelate,
+            }
+        return result
+
+def clear_views_without_print_report(self, view_id, resreport):
```
Comment (rimbendhaou) [inline project_event/models/models.py:93]: PEP8 expected two blank lines, found1

File: project_event/models/models.py [hunk 4]
```diff
@@ -21,3 +23,68 @@ def get_formview_id(self, access_uid=None):
...
+    views_without_print_report = [
+        'project_event.project_event_task_tree',
+        'project_event.project_event_task_form',
+        ]
+    if  view_id and self.env['ir.ui.view'].browse(view_id).get_xml_id()[view_id] in views_without_print_report:
```
Comment (rimbendhaou) [inline project_event/models/models.py]: double spaces after if (keep only one space)

File: project_event/models/models.py [hunk 5]
```diff
@@ -21,3 +23,68 @@ def get_formview_id(self, access_uid=None):
...
+        resreport = []
+    else:
+        return resreport
+
+BaseModel.fields_view_get = fields_view_get
```
Comment (rimbendhaou) [inline project_event/models/models.py:105]: PEP8 expected two blank lines, found1

**Review by rimbendhaou — CHANGES_REQUESTED** [review]

File: project_event/models/models.py (hunk 2 above)
Comment (eilst) [reply project_event/models/models.py:26]: Done

File: project_event/models/models.py (hunk 4 above)
Comment (eilst) [reply project_event/models/models.py]: Done

File: project_event/models/models.py (hunk 5 above)
Comment (eilst) [reply project_event/models/models.py:105]: done

File: project_event/report/project_task_activity_report.py (hunk 1 above)
Comment (eilst) [reply project_event/report/project_task_activity_report.py]: Because there will also be Contact 2, specs

**Review by rimbendhaou — APPROVED** [review]

**Pull request merged**